cradle-core/src/main/java/com/exactpro/cradle/testevents/TestEventsMessagesLinker.java

```diff
@@ -25,19 +25,38 @@
 public interface TestEventsMessagesLinker
 {
 	/**
+	 * This method is deprecated, use {@link #getTestEventIds(StoredMessageId) getTestEventIds}
 	 * Retrieves IDs of stored test events by linked message ID
 	 * @param messageId ID of stored message
 	 * @return collection of stored test event IDs
 	 * @throws IOException if test event data retrieval failed
 	 */
+	@Deprecated
 	Collection<StoredTestEventId> getTestEventIdsByMessageId(StoredMessageId messageId) throws IOException;
 	
 	/**
+	 * This method is deprecated, use {@link #getTestEventIdsAsync(StoredMessageId) getTestEventIdsAsync}

```

Comment:
There is a `@deprecated` option in JavaDoc. Maybe we should use that option instead

Reply:
Thank you for pointing this out, this way it is better.